source/content/workspace-offboarding.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Workspace Offboarding
+description: Easily remove users and supporting workspaces from multiple sites.
```

Comment:
since there are WordPress users and Drupal users in addition to the Pantheon team/account roles that could exist for a site, I suggest:

```suggestion
description: Easily remove Pantheon users and supporting workspaces from multiple Pantheon sites.
```